pkg/generated/clientset/internalversion/clientset.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 Red Hat, Inc.
+Copyright 2022 Red Hat, Inc.
```

Comment:
Consider adding to the commit message the command used to generate the files.

Reply:
this is generated code.